README.rst

```diff
@@ -43,7 +43,7 @@ be called ``ghostscript`` too.
 API
 ---
 
-``generate_barcode(barcode_type, data, options)``
+``generate_barcode(barcode_type, data, options=None)``
 -------------------------------------------------
```

Comment:
the number of `-` should match the heading above in reStructuredText, that's the build failure:

```
Running setup.py check
running check
warning: check: Title underline too short. (line 47)
error: Please correct your package.
ERROR: InvocationError: '/home/travis/build/YPlan/treepoem/.tox/py36-codestyle/bin/multilint'
```